code/game/machinery/computer/abnormality_archive.dm

```diff
@@ -0,0 +1,89 @@
+/obj/machinery/computer/abnormality_archive
+	name = "abnormality archive"
+	desc = "A secure terminal connected to the Lobotomy Corp archive."
+	icon = 'icons/obj/computer.dmi'
+	icon_state = "computer"
+	var/printfile
+	var/printing_status
+	var/notehtml
+	var/note
+	var/list/note_list
+	var/linecount
+	var/paper = 1 //You get one paper
+	var/papermax = 10 //Recycle more paper
+
+/obj/machinery/computer/abnormality_archive/ui_interact(mob/user, datum/tgui/ui)
+	. = ..()
+	ui = SStgui.try_update_ui(user, src, ui)
+	if(!ui)
+		ui = new(user, src, "LC13AbnormalityArchive", name) //apparently connected to .Js which is invisable to dreammaker but not to Visual Studio Code.
+		ui.open()
+
+/obj/machinery/computer/abnormality_archive/ui_static_data(mob/user) //list is created the moment a person boots up the UI and inherited by future iterations of this object
+	. = ..()
+	var/abnormality_info
+	if(!abnormality_info)
+		abnormality_info = list()
+		for(var/_record_detail in subtypesof(/obj/item/paper/fluff/info)) //consists of all threat levels and their branches
+			var/obj/item/paper/fluff/info/record_detail = _record_detail
+			var/list/record_data = list()
+			if(initial(record_detail.name) == "paper" || initial(record_detail.noArchive) == TRUE) //removes template papers and allows for secret files
+				continue
+			note_list = splittext(initial(record_detail.info), "<br>") //splits text into a list by making <br> into a break,
+			popleft(note_list) //Remove first line since it is the name
+			pop(note_list) //Removes last line since it is almost always blank. If you are making your abnormality and encountering this issue i give you my blessing to remove this line.
+			linecount = note_list.len //Doing it seperately results in only half of the lines being given a value. A single variable seems to keep it stable.
+			if(!linecount)
+				return
+			record_data["linecount"] = linecount //len is list length
+			record_data["type"] = record_detail //gives the typepath of the file we are reading
+			record_data["name"] = remove_paper_commands(initial(record_detail.name))
+			for(var/i=0,i <= linecount,i++) //loop until we reach the same length as the linecount
+				record_data["line[i]"] += remove_paper_commands(popleft(note_list))
```

Comment:
i am vary about the execution order of the popleft vs the proc vs the addition but if it's tested and it works...